services/QuillLMS/client/app/bundles/Shared/libs/titleCase.ts

```diff
@@ -0,0 +1,3 @@
+export function titleCase(string: string){
+  return string[0].toUpperCase() + string.slice(1).toLowerCase();
```

Comment:
Maybe add an edge case here for when string is null or empty string? I know it's probably rare but if we end up using this method later down the line for other functions we might want it to be fail-safe.